deploy/ansible/playbooks/iroha-docker-cluster/iroha-deploy.yml

```diff
@@ -0,0 +1,16 @@
+  - hosts: locals
+    roles:
+      - { role: iroha-cluster-config-gen }
```

Comment:
Also, in iroha/deploy/ansible/playbooks/iroha-docker-cluster/iroha-deploy.yml it is not really convenient to regenerate configs each time, and it takes a lot of time, can you please make a parameter or something so that iroha-cluster-config-gen role is executed only once if needed, and then we may relaunch docker network if needed?

Reply:
@neewy it is rather easier to comment config file generation role. regarding file delivery - `tags` could be used. i'll provide description in the next commit

Reply:
I was always commenting it out, it was not very convenient really. Is it possible to run some tasks or not run depending on the CLI argument?

Reply:
@neewy yes, with usage of 'tags'